pages.es/osx/rubocop.md

```diff
@@ -0,0 +1,31 @@
+# rubocop
+
+> Analiza archivos de Ruby.
+
+- Verifica todos los archivos en el directorio actual (incluyendo subdirectorios):
+
+`rubocop`
+
+- Verifica uno o más archivos o directorios específicos:
+
+`rubocop {{path/to/file}} {{path/to/directory}}`
+
+- Guarda el output en un archivo:
```

Comment:
I think `salida` is the word because `salida estándar` is valid in Spanish, or `resultado`:
```suggestion
- Guarda la salida en un archivo:
```